pandas/core/generic.py

```diff
@@ -100,10 +101,22 @@ def __init__(self, data, axes=None, copy=False, dtype=None,
                 for i, ax in enumerate(axes):
                     data = data.reindex_axis(ax, axis=i)
 
-        object.__setattr__(self, 'is_copy', None)
+        object.__setattr__(self, '_parent', [])
         object.__setattr__(self, '_data', data)
         object.__setattr__(self, '_item_cache', {})
 
+    def _get_is_copy(self):
+        warnings.warn("is_copy is deprecated will be removed in a future release",
+                      FutureWarning)
+        return None
```

Comment:
Not sure, but if this *really* *internal* property is basically now meaningless, then IMO it would be better to cleanly break code which depends on this (=remove the property without warning) than just change the API contract of this property...